agent/lib/kontena/websocket_client.rb

```diff
@@ -101,26 +100,27 @@ def send_message(msg)
       error "failed to send message: #{exc.message}"
     end
 
+    def send_notification(method, params)
```

Comment:
Probably not because I did cut this from bigger refactoring... That refactor used this method instead of the custom message format that `QueueWorker` currently uses. I'll remove it from here.